saleor/core/permissions.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib.auth.models import Permission
+
+
+GROUP_PERMISSIONS_MODELS = [
+    'product.product',
+    'product.category',
+    'product.stock_location',
+    'discount.sale',
+    'discount.voucher',
+    'order.order',
+    'userprofile.user'
+]
+
+
+def get_user_groups(user):
+    return {'groups': list(user.groups.values_list('name', flat=True))}
+
+
+def get_permissions():
+    codenames = []
+    for group_permission in GROUP_PERMISSIONS_MODELS:
+        model_name = group_permission.split('.')[1]
+        codenames.append('view_%s' % model_name)
+        codenames.append('edit_%s' % model_name)
```

Comment:
Why can't we list those explicitly in `GROUP_PERMISSIONS`?

Reply:
We can but the idea was to edit only one place when add new permission to view/edit sth and to filter out those default from django.

Reply:
You will still edit only one place. You'll just need to add two permissions there. What if we introduce a financial reporting feature that does not use an "edit" permission or we need to limit the ability to delete certain items?

Reply:
This was consulted with @elwoodxblues. I think we should discus this together.